content/influxdb/cloud-dedicated/reference/internals/security.md

```diff
@@ -260,13 +262,49 @@ exchanged with `influxctl`.
 After a successful Auth0 authentication, {{% product-name %}} provides the
 user's `influxctl` session with a short-lived
 [management token](#management-tokens) for access to the Granite service.
-The user interacts with the `influxctl` command line tool to manage the workload
-cluster, including creating [database tokens](#database-tokens) for database
-read and write access and [creating long-lived management tokens](/influxdb/cloud-dedicated/admin/management-tokens/)
-for use with the [Management API](/influxdb/cloud-dedicated/api/management/).
+The user interacts with the `influxctl` command line tool to view or manage
+cluster resources.
+The [user groups](#user-groups) assigned to the user determine the level of
+access to resources.
+
+#### User groups
+
+User groups associate access privileges with user attributes--an important part of the
+Attribute-Based Access Control (ABAC) security model, which grants access based on
+user attributes, resource types, and environment context.
+
+In {{% product-name %}}, a user can belong to any of the following user groups,
+each with predefined privileges:
+
+- [Admin user group](#admins-role-admin)
+- [Member user group](#members-role-member)
+- [Auditor user group](#auditors-role-auditor)
+
+##### Admins (role: admin)
```

Comment:
@dsprogis Splitting hairs here, but are we calling these roles or groups? I guess I ask because I know this lays the ground work for ABAC, but "roles" pushes it towards RBAC.

Reply:
> @dsprogis Splitting hairs here, but are we calling these roles or groups? I guess I ask because I know this lays the ground work for ABAC, but "roles" pushes it towards RBAC.

To be clear, the use of "role" here was my uninformed choice, just repeating what I saw in v1. My understanding though is groups essentially represent roles--the product brief says "role-based"-- but are stored as user attributes.

Reply:
I think Scott is right that we are overloading the term "role" in proximity to RBAC.
In context, perhaps the word "responsibilities" works better?  (..."users have “management” responsibilities, such as creating...)